js_modules/dagit/packages/core/src/graph/OpTags.tsx

```diff
@@ -39,7 +38,13 @@ function hueForTag(text = '') {
 
 function getTag(tag: IOpTag) {
   if (tag.label === 'noteable') {
-    return <Icon name="noteable_logo" />;
+    return <img src={noteable_logo} />;
```

Comment:
Maybe before we merge, want to add `alt="Noteable logo" role="img"` to each of these? Tthe linter isn't complaining but I thought that it would...